code/modules/power/singularity/singularity.dm

```diff
@@ -60,6 +60,19 @@
 	consume(user)
 	return 1
 
+/obj/singularity/attack_paw(mob/user)
+	consume(user)
+
+/obj/singularity/attack_alien(mob/user)
+	consume(user)
+
+/obj/singularity/attack_animal(mob/user)
+	consume(user)
+
+/obj/singularity/attackby(obj/item/weapon/W, mob/user, params)
+	consume(user)
+	return 1
```

Comment:
"Urist has attacked the singularity with a toolbox"
Believe it or not it actually works, I tested it, lol.